main.go

```diff
@@ -52,6 +55,20 @@ func getConfig() (*config, error) {
 	gitlabToken := os.Getenv("GITLAB_TOKEN")
 	gitlabBaseURL := os.Getenv("GITLAB_BASE_URL")
 	integrationDirectory := os.Getenv("INTEGRATION_DIRECTORY")
+	osToEntGithubToken  := os.Getenv("OS_SYNC_GITHUB_TOKEN")
```

Comment:
Why do we need a new token? It should work with `GITHUB_TOKEN`

Reply:
Yeh, I figured this would come up. I guess it is not necessary with a restricted token anymore, so remove and default to the old one?

Reply:
yes, please